schemas/telemetry/core/core.2.schema.json

```diff
@@ -32,6 +32,12 @@
     "arch": {
       "type": "string"
     },
+    "experiments": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
```

Comment:
Now sorted to make future artifact compares easier... not going to update the master and reconstruct this PR again since it only impacts only a few small diffs.